ubuntudesign/developer.ubuntu.com
# Add canonical-global-nav as a dependency

## Done

Add canonical-global-nav as a dependency

## QA

- Run 'npm install -g bower'
- Run 'bower install canonical-global-nav'
- Run './run'
- Pop on over to http://0.0.0.0:8015 and make sure the global nav is visible
- Check in l, m and s
